### PR TITLE
Initial policy implementation

### PR DESCRIPTION
**Pull request by flavio**
The policy works as expected. Only e2e tests are missing

I'll push them during the next week

### CODE REVIEW

File: src/custom_resources.rs [hunk 1]
```diff
@@ -0,0 +1,190 @@
+use k8s_openapi::apimachinery::pkg::api::resource::Quantity;
+use k8s_openapi::schemars;
+use serde;
```
Comment (jvanz) [inline src/custom_resources.rs:3]:
```suggestion
```

Comment (jvanz) [reply src/custom_resources.rs:3]: I've tried to applied this fix. But I cannot.

**Review by jvanz — COMMENTED** [review]
LGTM. But we still need to add the E2E tests. :-)

**Comment by flavio** [comment]
> LGTM. But we still need to add the E2E tests. :-)

I'll add them with a dedicated PR

**Pull request merged**